core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -26,6 +26,7 @@ import java.util.{Date, Locale}
 import scala.collection.JavaConverters._
 import scala.collection.mutable
 import scala.collection.mutable.HashMap
+import scala.language.existentials
```

Comment:
for fix compile error of

```
[error] /core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala:575:36: inferred existential type org.apache.hadoop.fs.FSDataOutputStreamBuilder[S,B]( forSome { type S <: org.apache.hadoop.fs.FSDataOutputStream; type B <: org.apache.hadoop.fs.FSDataOutputStreamBuilder[S,B] }), which cannot be expressed by wildcards,  should be enabled
[error] by making the implicit value scala.language.existentials visible.
[error] ----
[error] This can be achieved by adding the import clause 'import scala.language.existentials'
[error] or by setting the compiler option -language:existentials.
[error] See the Scaladoc for value scala.language.existentials for a discussion
[error] why the feature should be explicitly enabled.
[error]         val builder = fs.createFile(qualifiedPath)
[error]                                    ^
[error] one error found
[error] (core / Compile / compileIncremental) Compilation failed
[error] Total time: 19 s, completed 2023-5-12 10:05:06
```